stl/inc/vector

```diff
@@ -510,18 +655,21 @@ public:
 
     _CONSTEXPR20 vector(initializer_list<_Ty> _Ilist, const _Alloc& _Al = _Alloc())
         : _Mypair(_One_then_variadic_args_t{}, _Al) {
+        _Asan_annotater<_Has_minimum_allocation_alignment<vector>> _Annotater(*this);

```

Comment:
Can we move the Annotation into `_Construct_n`

One of the reasons for it was to bundle logic there

Reply:
I can take a look at whether this works but I'm hesitant to do it.